src/ansys/fluent/visualization/__init__.py

```diff
@@ -61,5 +62,152 @@ def version_info() -> str:
 )
 from ansys.fluent.visualization.graphics import Graphics  # noqa: F401
 from ansys.fluent.visualization.plotter import Plots  # noqa: F401
-from ansys.fluent.visualization.registrar import get_renderer, register_renderer
+from ansys.fluent.visualization.registrar import (
+    _visualizer,
+    get_renderer,
+    register_renderer,
+)
 from ansys.fluent.visualization.renderer import GraphicsWindow
+
+
+class View(str, Enum):
+    """Available view options."""
+
+    XY = "xy"
+    XZ = "xz"
+    YX = "yx"
+    YZ = "yz"
+    ZX = "zx"
+    ZY = "zy"
+    ISOMETRIC = "isometric"
+
+
+class _ViewWrapper:
+    def __init__(self, default=View.ISOMETRIC):
+        self._value = default
+
+    def __call__(self):
+        return self._value
+
+    def __str__(self):
+        return str(self._value)
+
+    def __repr__(self):
+        return repr(self._value)
+
+    def __eq__(self, other):
+        return self._value == other
+
+    def __getattr__(self, attr):
+        # Allows access like config.view.name or config.view.value
+        return getattr(self._value, attr)
+
+    def allowed_values(self):
+        """List of allowed views."""
+        return list(View)
+
+    def set(self, val):
+        """Set view."""
+        if isinstance(val, str):
+            val = View(val)
+        elif not isinstance(val, View):
+            raise TypeError("view must be of type 'View' or str")
+        self._value = val
+
+
+class Config:
+    """Set the configuration variables for visualization."""
+
+    def __init__(self):
+        """__init__ method of Config class."""
+        self._interactive = True
+        self._view = _ViewWrapper()
+        self._single_window = False
+        self._two_dimensional_renderer = "pyvista"
+        self._three_dimensional_renderer = "pyvista"
+
+    @property
+    def interactive(self):
+        """Boolean flag to access mode (interactive or non-interactive)."""
+        return self._interactive
+
+    @interactive.setter
+    def interactive(self, val: bool):
+        """Set mode (interactive or non-interactive)."""
+        self._interactive = bool(val)
+
+    @property
+    def single_window(self):
+        """Whether single Qt window is activated."""
+        return self._single_window
+
+    @single_window.setter
+    def single_window(self, val: bool):
+        """Activate (or Deactivate) single Qt window."""
+        self._single_window = bool(val)
+
+    @property
+    def view(self):
+        """Returns the camera angle set for displaying graphics."""
+        return self._view
+
+    @view.setter
+    def view(self, val):
+        """Sets the camera angle set for displaying graphics."""
```

Comment:
@prmukherj This and the docstring of the method above both imply that the user sets and gets back a number signifying an angle in some units and carrying meaning.

Reply:
Done.